tardis/model/base.py

```diff
@@ -189,6 +191,33 @@ def abundance(self):
         abundance.columns = range(len(abundance.columns))
         return abundance
 
+    def decay(self, day, normalize=True):
```

Comment:
I think this should happen during the initialization of the model.

The behavior should be:
If there is `isotope_abundance` as parameter, decay these with `time_explosion` and merge them with the normal abundances. If we do this, save the normal abundances and the undecayed abundances somewhere on the model (example `raw_abundance`, `raw_isotope_abundance`)